graphql_compiler/tests/test_helpers.py

```diff
@@ -591,14 +593,15 @@ def get_type_equivalence_hints() -> TypeEquivalenceHintsType:
     return type_equivalence_hints
 
 
+@lru_cache()
 def get_common_schema_info() -> CommonSchemaInfo:
     """Get the default CommonSchemaInfo used for testing."""
     return CommonSchemaInfo(get_schema(), get_type_equivalence_hints())
 
 
 def _get_schema_without_list_valued_property_fields() -> GraphQLSchema:
     """Get the default testing schema, skipping any list-valued property fields it has."""
-    schema = get_schema()
+    schema = build_ast_schema(parse(SCHEMA_TEXT))
```

Comment:
This was necessary because this function modifies the schema stored in the `schema` variable.